examples/if_this_then_that/common/__init__.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2016 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''If This Then That helper common code.
+
+This module include a subclass of Robot to add some helper methods that are
+useful to the IFTTT examples.
+
+This class includes the following:
+    1) get_in_position moves Cozmo's lift down and face up if necessary.
+    2) display_image_on_face displays the requested image file on his face after
+        the If This Then That trigger has been received.
+'''
+
+import sys
+
+import cozmo
+
+
+try:
+    from PIL import Image, ImageDraw, ImageFont
+except ImportError:
+    sys.exit("Cannot import from PIL: Do `pip3 install Pillow` to install")
```

Comment:
Can you change this to the following (I think i updated all the other examples to add "--user" after you started working on this):
sys.exit("Cannot import from PIL. Do `pip3 install --user Pillow` to install")